components/playlist/browser/playlist_service.h

```diff
@@ -95,6 +96,9 @@ class PlaylistService : public KeyedService,
   PlaylistService(const PlaylistService&) = delete;
   PlaylistService& operator=(const PlaylistService&) = delete;
 
+  mojo::PendingRemote<mojom::PlaylistService> MakeRemote();
+  void Bind(mojo::PendingReceiver<mojom::PlaylistService> receiver);
```

Comment:
I'm wondering if this would be called automatically by the framework, because I couldn't find who is calling this.